js/templates/modals/settings/advanced.html

```diff
@@ -213,30 +221,35 @@ <h2 class="h4 clrT"><%= ob.polyT('settings.advancedTab.smtp._sectionName') %></h
               <label for="smtpServerAddress">
                 <%= ob.polyT('settings.advancedTab.smtp.serverAddress') %>
               </label>
+              <div class="clrT2 txBase"><%= ob.polyT('settings.advancedTab.smtp.helperServerAddress') %></div>
             </div>
-            <div class="col9">
+            <div class="col6">
               <% if (ob.errors['smtpSettings.serverAddress']) print(ob.formErrorTmpl({ errors: ob.errors['smtpSettings.serverAddress'] })) %>
-              <input class="clrBr" 
+              <input class="clrBr clrSh2" 
                 required
                 type="text"
                 name="smtpSettings.serverAddress" 
                 id="smtpServerAddress" 
+                placeholder="<%= ob.polyT('settings.advancedTab.smtp.placeholderServerAddress') %>"
                 value="<%= ob.smtpSettings.serverAddress %>">
+              <div class="clrT2 txBase padSm"><%= ob.polyT('settings.advancedTab.smtp.helperNewEmail') %> <a href="https://accounts.google.com/SignUp" class="clrTEm">Gmail.com</a></div>
```

Comment:
I think it's quite risky to expose html to translators (especially with our classes in there). I've been doing something like this, which allows the translators to position the link, but doesn't expose the HTML to them:

```
"gmailLink": "Gmail.com",
"helperNewEmail": "Create an email address at %{gmailLink}",
```

then in the template:

 <a href="https://accounts.google.com/SignUp" class="clrTEm">Gmail.com</a>

```
<div class="clrT2 txBase padSm"><%= ob.polyT('settings.advancedTab.smtp.helperNewEmail', { gmailLink: `<a href="https://accounts.google.com/SignUp" class="clrTEm">${ob.polyT('settings.advancedTab.smtp.gmailLink')}</a>` }) %></div>
```